magefile.go

```diff
@@ -902,8 +912,11 @@ func fetchBinaryFromArtifactsApi(ctx context.Context, packageName, artifact, ver
 		false,
 		downloadPath,
 		true)
-	fmt.Println("downloaded binaries on location:", location)
+	if err != nil {
+		return err
+	}
 
+	fmt.Println("downloaded binaries on location:", location)
```

Comment:
[Suggestion]
It feels a bit redundant
```suggestion
	fmt.Println("downloaded binaries on", location)
```